ouroboros-consensus/docs/utxo-hd/report.tex

```diff
@@ -0,0 +1,741 @@
+only provides partial information. The data has then been split between the
+\emph{in-memory} part and the \emph{on-disk} part.
+
+The core of the UTxO-HD implementation are what we call \emph{the ledger
+  tables}.
+
+\begin{lstlisting}
+data family LedgerTables l :: LedgerStateKind
+\end{lstlisting}
+
+A ledger state now has ledger tables attached to it, which live
+outside the ledger state. The ledger tables represent partial information for
+the maps that have been moved to disk, in particular as mentioned above in this
+version only the UTxO set.
+
+\begin{figure}[h]
+  \centering
+  \begin{tikzpicture}
+    \filldraw[fill=blue!40!white, draw=black] (0,0) rectangle (2,2);
+    \filldraw[fill=white, draw=black] (1.2,0.8) circle (0.7);
+    \node at (1,2.5) {LedgerState};
+    \filldraw[fill=blue!40!white, draw=black] (4,0.8) circle (0.7);
+    \node at (4,2) {LedgerTables};
+  \end{tikzpicture}
+  \caption{The separation between the ledger state and the tables}
+\end{figure}
+
+Note that the hole left by extracting the tables is not really a hole, but
+instead it is filled with an empty \texttt{Data.Map} container.
+
+The tables themselves can represent different logical states of the UTxO set,
+either actual values, differences, both or even represent no information about
+the UTxO set at all.
+
+\begin{lstlisting}
+data MapKind :: MapKind' -> Type -> Type -> Type where
+  EmptyMK    ::                                  ApplyMapKind' EmptyMK'    k v
+  KeysMK     :: !(Keys    k v)                -> ApplyMapKind' KeysMK'     k v
+  ValuesMK   :: !(Values  k v)                -> ApplyMapKind' ValuesMK'   k v
+  DiffMK     :: !(Diff    k v)                -> ApplyMapKind' DiffMK'     k v
+  SeqDiffMK  :: !(SeqDiff k v)                -> ApplyMapKind' SeqDiffMK'  k v
+  TrackingMK :: !(Values  k v) -> !(Diff k v) -> ApplyMapKind' TrackingMK' k v
+\end{lstlisting}
+
+The ledger tables of \emph{values} are \emph{partial projections} of data which
+currently can be extracted from a ledger state so naturally we have functions to
+inject or project.
+
+\begin{lstlisting}
+class StowableLedgerTables l where
+  stowLedgerTables   :: l ValuesMK -> l EmptyMK
+  unstowLedgerTables :: l EmptyMK  -> l ValuesMK
+\end{lstlisting}
+
+\emph{Differences} are no more than specifications of keys and values that have been
+either inserted or deleted when going from one ledger state to another. In particular, differences can be applied to values, process which we will call \textbf{forwarding}.
+
+\begin{observation}
+  \label{ko1}
+  \emph{The only way to mimick a full ledger state is by holding a initial set
+    of values and a sequence of differences}.
+\end{observation}
+
+\subsubsection{Notation}
+When representing a ledger state from now on, in case the ledger state is as
+known by the ledger (oblivious to ledger tables), we will denote it with
+$\mathbb{L}$, and otherwise we will denote it as $L^{s}_{T}$ where $s$ means the
```

Comment:
The notation has been redefined more or less following this.

About your question, $L$ is well-defined because it is the current tip. $L^s$ is just $L$ ticked to slot $s$